assets/admin/editor-wizard/steps/lesson-details-step.js

```diff
@@ -44,7 +40,7 @@ const LessonDetailsStep = ( { data: wizardData, setData: setWizardData } ) => {
 				<div className="sensei-editor-wizard-step__form">
 					<LimitedTextControl
 						label={ __( 'Lesson Title', 'sensei-lms' ) }
-						value={ wizardData.lessonTitle ?? '' }
+						value={ wizardData.lessonTitle ?? postTitle }
```

Comment:
I think this check could also be part of the hook, so it handles all title stuff. WDYT?

It would also be nice if we could unify the variable name. Maybe use `lessonTitle` in all the places (in the hook return and while calling the hook)?

Reply:
You are a genius! I couldn't find a better name because I had this check here. I dind't think of moving this also inside the hook! THANKS!